extensions/PhabBugz/lib/WebService.pm

```diff
@@ -40,6 +41,8 @@ use constant PUBLIC_METHODS => qw(
     revision
 );
 
+use constant PHAB_BMO_USER_EMAIL => 'phab-bot@bmo.tld';
```

Comment:
if you rebase (once #166 lands soon), you can use PHAB_AUTOMATION_USER in Constants.pm now.

use Bugzilla::Extension::PhabBugz::Constants;